src/cloudflare/workers.ts

```diff
@@ -91,3 +92,28 @@ export const env = new Proxy(
 );
 
 export const waitUntil = entrypoints.waitUntil.bind(entrypoints);
+
+export function nodeCompatHttpServerHandler({
```

Comment:
Really important that there is an equivalent that returns just the fetch implementation
We should have something to support adding JSRPC methods to the entrypoint
```
export default {
  nodeCompatHttpServerHandlerFetch(port),
  async scheduled(...) {}
  async queue(...) {}
}
```

Reply:
One can do

```js
const handler = nodeCompatHttpServerHandler({port});
handler.queue = async queue(...) {};
export default handler;
```

Reply:
Either this or something like...

```
export default nodeCompatHttpServerHandlerFetch(port, {
  async scheduled(...) {},
  async queue(...) {}
});
```

This would align more with the model hono has pioneered here.. https://github.com/honojs/examples/blob/main/durable-objects/src/index.ts (e.g. `export default app`)

Reply:
I'll keep this conversation unresolved, so we don't forget to add them to docs. Thanks for the reminder @danlapid